api/src/components/reference/controller.ts

```diff
@@ -53,9 +137,26 @@ export const remove = async (event: I.AuthenticatedAPIRequest<undefined, undefin
     }
 };
 
-export const removeAll = async (event: I.AuthenticatedAPIRequest<undefined, undefined, I.RemoveAllReferencesPath>) => {
+export const removeAll = async (
```

Comment:
we can remove this also, because the updateAll takes care of deleting all the references for a specific publication